Dockerfile

```diff
@@ -136,6 +139,23 @@ COPY scripts/async_profiler_build_shared.sh .
 COPY scripts/async_profiler_build_musl.sh .
 RUN ./async_profiler_build_shared.sh /tmp/async_profiler_build_musl.sh
 
+# node-package-builder-musl
+FROM alpine${NODE_PACKAGE_BUILDER_MUSL} AS node-package-builder-musl
+WORKDIR /tmp
+COPY scripts/build_node_package.sh .
```

Comment:
Place it right before the `RUN ./build_node_package.sh` - this `COPY` stage is very cheap, while the next `RUN` stage is not, as it installs many packages. If we change the script, we don't need the `apk add` to run again.

Reply:
Ping @kwisniewski98 :)

Reply:
Ah done